readme.md

```diff
@@ -180,7 +185,11 @@ customer or a charge, but only used once.
 	var tokenService = new StripeTokenService();
 	StripeToken stripeToken = tokenService.Create(myToken);
 
-Tokens are very useful if you don't want to store the customers credit card information on your servers (pci compliance).
+You should never create tokens with Stripe.net with real credit cards, since
+giving your server access to unredacted card data greatly increases your
+PCI-DSS compliance burden.
```

Comment:
My assumption was the same, probably to a fault given the article you just posted (I did not see that). And my intention was for it never to come across that you were compliant simply by generating a token on your server. This is something in the works for Stripe.net to make this process easier (interacting with client side). Given your explanation of why this is a problem now, I'm good with your new wording here.